embedded_gems/pathogen/pathogen_view_components.gemspec

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pathogen/view_components/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'pathogen_view_components'
+  spec.authors       = ['Bioinformatics and Computational Biology Unit, Public Health Agency of Canada']
+  spec.email         = ['bioinformatics@phac-aspc.gc.ca']
```

Comment:
**nitpick (blocking):** Can you leave email out of here, since that is not a valid email.

Reply:
Removed in 19d951acda9c760ef2cf3a1d6dc8a404048aba86